File: articles/evaluation-benchmarks-and-the-limits-of-ai-measurement/fortran/evaluation_metric.f90
program evaluation_metric
  implicit none
  real :: tp, tn, fp, fn, accuracy
  tp = 42.0
  tn = 38.0
  fp = 7.0
  fn = 13.0
  accuracy = (tp + tn) / (tp + tn + fp + fn)
  print *, "accuracy=", accuracy
end program evaluation_metric

File: articles/evaluation-benchmarks-and-the-limits-of-ai-measurement/rust/evaluation_metric.rs
fn main() {
    let tp = 42.0;
    let tn = 38.0;
    let fp = 7.0;
    let fn_ = 13.0;
    let accuracy = (tp + tn) / (tp + tn + fp + fn_);
    println!("accuracy={:.4}", accuracy);
}
